include/simdutf/implementation.h

```diff
@@ -1765,6 +1781,19 @@ class implementation {
    */
     simdutf_warn_unused virtual size_t latin1_length_from_utf32(const char32_t * input, size_t length) const noexcept = 0;
 
+
+  /**
+   * Compute the number of bytes that this UTF-8 string would require in Latin1 format.
+   *
+   * This function does not validate the input.
```

Comment:
You don't really need to say that we don't validate because, as far as I know, all inputs are valid when you have latin 1.